event-sourcing/event-repository/event-repository-liquibase/src/main/resources/liquibase/event-store-db-changesets/018-create-linkeded-event-table.xml

```diff
@@ -4,7 +4,7 @@
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
          http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
 
-    <changeSet id="event-store-010" author="TechPod"
+    <changeSet id="event-store-018" author="TechPod"
```

Comment:
won't this make the change set have a different file signature
we should add the trick here to say this is an acceptable change